platform/darwin/src/MGLGeometry_Private.h

```diff
@@ -128,3 +128,5 @@ MGLRadianCoordinate2D MGLRadianCoordinateAtDistanceFacingDirection(MGLRadianCoor
 CLLocationDirection MGLDirectionBetweenCoordinates(CLLocationCoordinate2D firstCoordinate, CLLocationCoordinate2D secondCoordinate);
 
 CGPoint MGLPointRounded(CGPoint point);
+
+MGLMapPoint MGLMapPointMake(mbgl::Point<double> coord, double zoomLevel);
```

Comment:
To avoid conflicting with the `MGLMapPointMake()` in the public header, this method should be called `MGLMapPointFromPoint()`.